### DISCUSSION TITLE
Can i trigger the q-uploader 'upload' method outside the tag?

### DISCUSSION THREAD

**Discussion by OleksandrYastrebov** [quasarframework/quasar#13448] [General - Components / Directives / etc] [answered]
I'm trying to implement the account avatar update.
I would like to have the next behaviour:
When i click on the q-avatar - the 'upload method' in q-uploader should be triggered.
Any ideas how to make it happen?

**Comment by MilosPaunovic** [comment, marked as answer]
Add `ref` to uploader and call it's method when you click on avatar image.

**Reply by OleksandrYastrebov** [reply]
Thanks :)
